diagrams/transfer_cancelled_by_payer.puml

```diff
@@ -0,0 +1,7 @@
+@startuml
+UserA -> "Clearable Token": orderTransfer(to: UserB)
+"Clearable Token" -> Agent: ClearableTransferOrdered
+UserA -> "Clearable Token": cancelTransfer
+"Clearable Token" -> UserA: release held amount
+"Clearable Token" -> UserA: ClearableTransferCancelled
```

Comment:
I think the arrow with the event should go to Agent instead of UserA. UserA does the cancellation therefore I would say the agent needs to be informed.

Reply:
You're right!